micronaut-camunda-bpm-feature/src/test/kotlin/info/novatec/micronaut/camunda/bpm/feature/test/MnProcessEngineConfigurationTest.kt

```diff
@@ -0,0 +1,114 @@
+package info.novatec.micronaut.camunda.bpm.feature.test
+
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import io.micronaut.transaction.SynchronousTransactionManager
+import io.micronaut.transaction.TransactionCallback
+import io.micronaut.transaction.TransactionStatus
+import org.camunda.bpm.engine.HistoryService
+import org.camunda.bpm.engine.RuntimeService
+import org.camunda.bpm.engine.history.HistoricProcessInstance
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import java.sql.Connection
+import javax.inject.Inject
+
+/**
+ * Executes the inherited tests with a transaction manager.
+ *
+ * @author Lukasz Frankowski
+ * @author Tobias Schäfer
+ */
+@MicronautTest(transactional = false)
+open class MnProcessEngineConfigurationTest {
+    @Inject
+    lateinit var runtimeService: RuntimeService
+
+    @Inject
+    lateinit var historyService: HistoryService
+
+    @Inject
+    lateinit var transactionManager: SynchronousTransactionManager<Connection>
+
+    @Test
+    open fun testCommit() {
```

Comment:
do they need to be open?

Reply:
yes, I think so. We have a subclass inheriting from this class. If I remove "open" I get the error "is final and cannot be overwritten".